testcases/connectivity/android/android.robot

```diff
@@ -387,28 +412,34 @@ Start AVD
     END
     Sleep    300
 
-Close All Apps
-    Close appium server
-    Close AVD
+Shutdown All Test Services
+    Shutdown appium server
+    Shutdown AVD
 
-Close appium server
-    Log To Console    Close appium server
+Shutdown appium server
+    Log To Console    Shutdown appium server
     IF    '${os}' == 'Windows'
         Run Process    cmd.exe /c taskkill /F /IM node.exe    shell=True
```

Comment:
consider to update this step to stop the appium server for Windows when using wrapper `appium.bat` file